autocomplete.js

```diff
@@ -22,12 +22,21 @@ async function getInstanceTypes(query, params, client, region) {
 
 async function listSubnets(query, params, client) {
   const subnets = await client.describeSubnets().promise();
+  const createSubnetText = (subnet) => {
```

Comment:
Is there a reason this isn't a function declaration?

Reply:
I wanted this function to be local because it is not used anywhere else, so I declared it using an arrow function, If you'd like I can export this to **helpers.js** file.

Reply:
Putting it in the same file as a separate function doesn't pollute the `listSubnets` content. We don't need to ba aware of it's implementation details here, and we could always scroll down to check it.
Also this creates a new `createSubnetText` function at each `listSubnets` call.

Reply:
You are right, I exported this function to the **helpers.js** file.